stream-chat-android-offline/src/main/java/io/getstream/chat/android/offline/channel/ChannelController.kt

```diff
@@ -689,6 +695,19 @@ public class ChannelController internal constructor(
         }
     }
 
+    private fun updateAttachmentUploadState(messageId: String, uploadId: String, newState: Attachment.UploadState) {
+        val message = _messages.value[messageId]
+        if (message != null) {
+            val attachment = message.attachments.firstOrNull { it.uploadId == uploadId }
+            if (attachment != null) {
+                val newAttachments = message.attachments - attachment + attachment.copy(uploadState = newState)
```

Comment:
Will this change the order of attachments, always putting the one that we've just updated to the end of the list?